packages/cli/commands/import/credentials.ts

```diff
@@ -1,3 +1,9 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
we don't want to add file wide rule disablings anymore but rather remove them. We should either remove them and refactor or think if the rule is needed or should be changed.